chart/openfaas/templates/prometheus-cfg.yaml

```diff
@@ -26,13 +26,31 @@ data:
         static_configs:
           - targets: ['localhost:9090']
 
-      - job_name: "gateway"
+      - job_name: 'kubernetes-pods'
         scrape_interval: 5s
-        dns_sd_configs:
-          - names: ['gateway.{{ .Release.Namespace }}']
-            port: 8080
-            type: A
-            refresh_interval: 5s
+        honor_labels: false
+        kubernetes_sd_configs:
```

Comment:
Thanks this looks pretty complicated compared to before. Do you have a source of where this comes from?

Reply:
Here it is https://github.com/prometheus/prometheus/blob/master/documentation/examples/prometheus-kubernetes.yml